build/android/src/main/java/net.minetest.minetest/MtNativeActivity.java

```diff
@@ -97,7 +97,7 @@ public int getDisplayHeight() {
 
 	@Override
 	protected void onActivityResult(int requestCode, int resultCode,
-									Intent data) {
+	                                Intent data) {
```

Comment:
This is Minetest code, hence the code style guidelines apply. Please use spaces to indent.